counterexamples/admins/locality/invalid-max-limit-iso-sub-country-code.json

```diff
@@ -0,0 +1,88 @@
+{
+    "type": "Feature",
+    "properties": {
+      "theme": "admins",
+      "type": "locality",
+      "version": 0,
+      "updateTime": "2023-02-22T23:55:01-08:00",
+      "subType": "administrativeLocality",
+      "localityType": "country",
+      "adminLevel": 1,
+      "isoSubCountryCode": "US-NYCC",
+      "defaultLanguage": "english",
+      "drivingSide": "right",
+      "contextId": "northAmericaId",
+      "names": {
+        "common": [
```

Comment:
Same comment on `names` - rinse, lather, and repeat throughout.

Reply:
resolved